spec/models/user_spec.rb

```diff
@@ -1,6 +1,186 @@
 require 'spec_helper'
 
 describe User do
+    context "rank and match users" do
+        before :each do
```

Comment:
Is all this information necessary to test user matching? It looks like this is just the seed file copy and pasted...